packages/logs/src/boot/startLogs.ts

```diff
@@ -51,6 +52,8 @@ export function startLogs(
         status: StatusType.error,
       },
     })
+    addTelemetryDebug('Error reported to customer', { 'error.message': error.message })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏the three added telemetry logs are identical. Maybe add the error source in the context? It could be logs / rum / replay

Reply:
I don't! `service` is fine to distinghuish between RUM and Logs, but not replay. If you think we don't need to run some analytics between RUM and Replay, we can certainly rely on the message to figure out what was the source of the log.

Reply:
> If you think we don't need to run some analytics between RUM and Replay, we can certainly rely on the message

That sounds fine for now, let's wait and see if we feel the need to add more context later